administration/src/util/generateHashFromCardDetails.ts

```diff
@@ -8,26 +8,43 @@ export interface CardDetails {
     cardType: CardActivateModel.CardType,
 }
 
+const cardDetailsToBinary = (cardDetails: CardDetails) => {
+    const fullNameWithoutZero = new TextEncoder().encode(cardDetails.fullName)
+    const binary = new Uint8Array(
+        8 // int64 expirationDate
```

Comment:
I think is would be good to define constants here. These can  then be reused in line 21 for example